src/item_factory.cpp

```diff
@@ -2202,8 +2202,10 @@ void Item_factory::load_basic_info( JsonObject &jo, itype &def, const std::strin
     load_slot_optional( def.brewable, jo, "brewable", src );
     load_slot_optional( def.fuel, jo, "fuel", src );
     load_slot_optional( def.relic_data, jo, "relic_data", src );
+
+    // optional gunmod slot may also specify mod data
     load_slot_optional( def.gunmod, jo, "gunmod_data", src );
-    load_slot_optional( def.mod, jo, "mod_data", src );
+    load_slot_optional( def.mod, jo, "gunmod_data", src );
```

Comment:
`load_gunmod()` Already calls both of these functions, as both are needed. This revert allows items which are not typed as GUNMOD to exhibit correct behavior.